### ISSUE TITLE
ValueError: Could not find the root folder of the project.

### ISSUE THREAD

**Issue by RainFung** [gventuri/pandas-ai#533]
### 🐛 Describe the bug

ValueError: Could not find the root folder of the project.

**Comment by yakuzadave** [comment]
I was able to fix this by creating "./pandasai.json" with the following:

```json

{
  "save_logs": true,
  "verbose": false,
  "enforce_privacy": false,
  "enable_cache": true,
  "use_error_correction_framework": true,
  "max_retries": 3,
  "custom_prompts": {},
  "save_charts": false,
  "save_charts_path": "exports/charts",
  "custom_whitelisted_dependencies": [],
  "middlewares": [],
  "callback": null,
  "llm": "OpenAI",
  "llm_options": null,
  "saved_dfs": []
}

```

> Had to do a bit of digging in the helpers to see how it was trying to identify the root of the project for this, but it works.

**Comment by gventuri** [comment]
@RainFung @yakuzadave thanks a lot for reporting. This bug has been fixed on 1.1.3!
